protocol/avdl/chat1/local.avdl

```diff
@@ -22,7 +22,10 @@ protocol local {
   }
 
   record MessageAttachment {
-    string path;
+    string storageType;  // where the attachment object is stored (i.e. s3)
```

Comment:
do you think we'll want to version the attachment structure?  What if version 2 is just to KBFS and therefore doesn't need a key?